go.mod

```diff
@@ -50,3 +51,5 @@ replace (
 	sigs.k8s.io/controller-runtime => sigs.k8s.io/controller-runtime v0.1.10
 	sigs.k8s.io/controller-tools => sigs.k8s.io/controller-tools v0.1.11-0.20190411181648-9d55346c2bde
 )
+
+go 1.13
```

Comment:
@DhritiShikhar curious what does this do?

Reply:
The golang version I use got added in the go.mod file. I have removed this file changes.